source/elements/oneDAL/source/algorithms/nearest_neighbors/knn_classification.rst

```diff
@@ -172,6 +171,14 @@ following steps:
       \quad 1 \leq j \leq m, \; 0 \leq l < c.
 
 
+   **Option 5: Super lite.** Do not say anything about conditional probability:
```

Comment:
I'd do this but tell something like this probability P_lj of class/label l given feature vector x^'_j or similar and then the expression
The step 3 would use the same notation